### ISSUE TITLE
Brew (macos bigsur - intel) install, returns linux unsatisifed libfuse

### ISSUE THREAD

**Issue by iangregsondev** [telepresenceio/telepresence#1688]
Hi,

I have upgraded brew and installed osx fuse as per instructions i.e.

```
❯ brew install osxfuse --cask
.
.
==> Changing ownership of paths required by osxfuse; your password may be necessary.
🍺  osxfuse was successfully installed!

```

and now trying

```
brew install datawire/blackbird/telepresence
```

and get

```
❯ brew install datawire/blackbird/telepresence
==> Installing telepresence from datawire/blackbird
libfuse: Linux is required for this software.
Error: An unsatisfied requirement failed this build.
```

Anybody managed to do this ?

I don't think I am missing a step?

Thanks

**Comment by rooflz** [comment]
I'm on bigsur + intel as well and am seeing the same issue.

**Comment by vals-productions** [comment]
I'm on Catalina 10.15.7 with the latest updates and also see this issue.

```
...
osxfuse was successfully installed!
```

and then:

```
% brew install datawire/blackbird/telepresence
==> Installing telepresence from datawire/blackbird
libfuse: Linux is required for this software.
Error: An unsatisfied requirement failed this build.
```

**Comment by timirnich** [comment]
Same issue here, on BigSur. Installing lates update now to rule out that it has something to do with that (doubt it though).

**Comment by markthebault** [comment]
Same issue here

**Comment by tigranbs** [comment]
Found a workaround with `brew install datawire/blackbird/telepresence --ignore-dependencies`, everything seems to be working except when I do `--method inject-tcp` it crashes with this error.
_not 100% sure if that's related_ 🤷‍♂️

```
Traceback (most recent call last):
  File "/usr/local/bin/telepresence/telepresence/cli.py", line 135, in crash_reporting
    yield
  File "/usr/local/bin/telepresence/telepresence/main.py", line 81, in main
    user_process = launch(
  File "/usr/local/bin/telepresence/telepresence/outbound/setup.py", line 64, in launch
    return launch_inject(runner_, command, socks_port, env)
  File "/usr/local/bin/telepresence/telepresence/outbound/local.py", line 120, in launch_inject
    torsocks_env = set_up_torsocks(runner, socks_port)
  File "/usr/local/bin/telepresence/telepresence/outbound/local.py", line 71, in set_up_torsocks
    raise RuntimeError("SOCKS network proxying failed to start...")
RuntimeError: SOCKS network proxying failed to start...


Here are the last few lines of the logfile (see /Users/tigranbs/projects/smartly-v1/telepresence.log for the complete logs):

  27.7 TEL | [103] exit -11 in 0.07 secs.
  27.8 TEL | [104] Running: torsocks python3 -c 'import socket; socket.socket().connect(('"'"'kubernetes.default'"'"', 443))'
  27.9 TEL | [104] exit -11 in 0.07 secs.
  28.0 TEL | [105] Running: torsocks python3 -c 'import socket; socket.socket().connect(('"'"'kubernetes.default'"'"', 443))'
  28.1 TEL | [105] exit -11 in 0.07 secs.
  28.2 TEL | [106] Running: torsocks python3 -c 'import socket; socket.socket().connect(('"'"'kubernetes.default'"'"', 443))'
  28.3 TEL | [106] exit -11 in 0.07 secs.
  28.4 TEL | [107] Running: torsocks python3 -c 'import socket; socket.socket().connect(('"'"'kubernetes.default'"'"', 443))'
  28.4 TEL | [107] exit -11 in 0.07 secs.
  28.5 TEL | [108] Running: torsocks python3 -c 'import socket; socket.socket().connect(('"'"'kubernetes.default'"'"', 443))'
  28.6 TEL | [108] exit -11 in 0.07 secs.
  28.6 TEL | END SPAN local.py:42(set_up_torsocks)   15.1s
```

**Comment by Arttuv** [comment]
Same issue here, macOS Big Sur 11.3, intel.

**Comment by HawkerOH** [comment]
Same issue here too, Catalina 10.15.7, Intel

**Comment by tschmidt64-SC** [comment]
+1, Big Sur 11.3, Intel

**Comment by magnusja** [comment]
Yeah same problem, Mojave....

**Comment by davidfloresv** [comment]
Updating Xcode tools and macFuse didn't work neither

**Comment by rdcox** [comment]
Reposting this response from #1654 because it got me unblocked:

> Installing macFUSE 4.1.0 and SSHFS 2.5.0 from https://osxfuse.github.io/ worked out fine for me.
https://github.com/telepresenceio/telepresence/issues/1654#issuecomment-829998470

Running Big Sur 11.2.3 I did the above and then `brew install datawire/blackbird/telepresence --ignore-dependencies` and things were working after that.

**Comment by Achieve777** [comment]
> Reposting this response from #1654 because it got me unblocked:
> 
> > Installing macFUSE 4.1.0 and SSHFS 2.5.0 from https://osxfuse.github.io/ worked out fine for me.
> > [#1654 (comment)](https://github.com/telepresenceio/telepresence/issues/1654#issuecomment-829998470)
> 
> Running Big Sur 11.2.3 I did the above and then `brew install datawire/blackbird/telepresence --ignore-dependencies` and things were working after that.

it's work well after I upgrade my mac into 11.3.1

**Comment by magnusja** [comment]
> > Reposting this response from #1654 because it got me unblocked:
> > > Installing macFUSE 4.1.0 and SSHFS 2.5.0 from https://osxfuse.github.io/ worked out fine for me.
> > > [#1654 (comment)](https://github.com/telepresenceio/telepresence/issues/1654#issuecomment-829998470)
> > 
> > 
> > Running Big Sur 11.2.3 I did the above and then `brew install datawire/blackbird/telepresence --ignore-dependencies` and things were working after that.
> 
> You can install it, but It does't work

See comment above, manually installing fuse and sshfs is essential.

**Comment by ronanknob** [comment]
> Reposting this response from #1654 because it got me unblocked:
> 
> > Installing macFUSE 4.1.0 and SSHFS 2.5.0 from https://osxfuse.github.io/ worked out fine for me.
> > [#1654 (comment)](https://github.com/telepresenceio/telepresence/issues/1654#issuecomment-829998470)
> 
> Running Big Sur 11.2.3 I did the above and then `brew install datawire/blackbird/telepresence --ignore-dependencies` and things were working after that.

Thanks, worked here in Big Sur 11.3.1 too!

**Comment by thallgren** [comment]
Closing this in favor of #1654 since I think it's better if we collect all brew install of sshfs/macFUSE in one ticket.
